### PR TITLE
Fix spelling and grammar mistakes

### PR DESCRIPTION
**Pull request by watameln**

### CODE REVIEW

File: Loader/Config/Settings.lua [hunk 1]
```diff
@@ -237,7 +237,7 @@ settings.PlayerPrefix = "!"			-- The ! in !donate; Mainly used for commands that
 settings.SpecialPrefix = ""			-- Used for things like "all", "me" and "others" (If changed to ! you would do :kill !me)
 settings.SplitKey = " "				-- The space in :kill me (eg if you change it to / :kill me would be :kill/me)
 settings.BatchKey = "|"				-- :kill me | :ff bob | :explode scel
-settings.ConsoleKeyCode = "Quote"	-- Keybind to open the console; Rebindable per player in userpanel; KeyCodes: https://developer.roblox.com/en-us/api-reference/enum/KeyCode
+settings.ConsoleKeyCode = "Quote"	-- Keybind to open the console; Rebindable per player in user-panel; KeyCodes: https://developer.roblox.com/en-us/api-reference/enum/KeyCode
```
Comment (Expertcoderz) [inline Loader/Config/Settings.lua:240]: It's either "userpanel" or "user panel". No hyphen.

File: Loader/Config/Settings.lua [hunk 2]
```diff
@@ -286,7 +286,7 @@ settings.Trello_Token = ""				-- Trello token (DON'T SHARE WITH ANYONE!)    Link
 settings.Trello_HideRanks = false		-- If true, Trello-assigned ranks won't be shown in the admins list UI (accessed via :admins)
 
 settings.G_API = true					-- If true allows other server scripts to access certain functions described in the API module through _G.Adonis
-settings.G_Access = false				-- If enabled allows other scripts to access Adonis using _G.Adonis.Access; Scripts will still be able to do things like _G.Adonis.CheckAdmin(player)
+settings.G_Access = false				-- If enabled allow other scripts to access Adonis using _G.Adonis.Access; Scripts will still be able to do things like _G.Adonis.CheckAdmin(player)
```
Comment (Expertcoderz) [inline Loader/Config/Settings.lua:289]: Broken grammar moment. I would suggest inserting a comma.

**Review by Expertcoderz — CHANGES_REQUESTED** [review]
Looks mostly good, just those few issues that need addressing.

**Comment by watameln** [comment]
my crap buggin'... marking stuff as things when I don't want to.

File: Loader/Config/Settings.lua [hunk 3]
```diff
@@ -137,7 +137,7 @@ local descs = {};			--// Contains settings descriptions
 			Card format refers to how card names should look
 
 
-			MAKE SURE YOU SET settings.DataStoreKey TO SOMETHING ABSOLUTELY RANDOM.
+			MAKE SURE YOU SET settings.DataStoreKey TO SOMETHING RANDOM.
```
[the remark this answers is not in the document]
Comment (watameln) [reply Loader/Config/Settings.lua:140]: Alright sounds good.

**Comment by watameln** [comment]
I think I've fixed those few issues that you listed.

**Comment by NoobBucket** [comment]
you might need to abandon your changes to Settings.lua since it will clash with my re-designing PR of it. i will change the mistakes you've found in my PR

**Comment by watameln** [comment]
Where can I revert it? Github is confusing

**Comment by Expertcoderz** [comment]
> you might need to abandon your changes to Settings.lua since it will clash with my re-designing PR of it. i will change the mistakes you've found in my PR

Eh isn't it the job of a maintainer to manually resolve the clashes when merging the PR? Don't ask people to take down their contributions in favor of yours just because they edited the same original file that you changed up in your PR; Git is built to allow for handling issues such as this in a cleaner fashion.

image

**Review by Expertcoderz — APPROVED** [review]

**Comment by NoobBucket** [comment]
> > you might need to abandon your changes to Settings.lua since it will clash with my re-designing PR of it. i will change the mistakes you've found in my PR
> 
> Eh isn't it the job of a maintainer to manually resolve the clashes when merging the PR? Don't ask people to take down their contributions in favor of yours just because they edited the same original file that you changed up in your PR; Git is built to allow for handling issues such as this in a cleaner fashion.
> 
> image

then i shall revert my changes

**Pull request merged**